src/platforms/javascript/guides/capacitor/troubleshooting.mdx

```diff
@@ -5,6 +5,38 @@ keywords: ["minimum version", "deployment target"]
 sidebar_order: 1000
 ---
 
+### Different versions for Sibling SDKs 
+
+Sentry Capacitor requires your app to use the same versions of @Sentry/Angular (or another Sibling SDK) that is refered by the peer dependency of Sentry Capacitor, to avoid any issues when installing packages or using the code on another machine, it's recommended to use the exact version of both sentry/capacitor and the sibling SDK. 
+
+Correct
+
+```xml
+    "@sentry/angular": "7.13.0",
+    "@sentry/capacitor": "0.10.1"
+```
+
+Incorrect
+
+```xml
+    "@sentry/angular": "^7.13.0",
+    "@sentry/capacitor": "^0.10.1"
+```
+
+One side effect of mismatching the SDKs is the follwing error during build time:
+
+```js
+  node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.
+```
+
+To solve this you must use the exact version of the Sibling SDK.
```

Comment:
You've already told them what the solution is, so I don't think you need it here.
```suggestion
```